hugegraph-mysql/src/main/java/com/baidu/hugegraph/backend/store/mysql/MysqlOptions.java

```diff
@@ -97,4 +97,12 @@ public static synchronized MysqlOptions instance() {
                     disallowEmpty(),
                     "disable"
             );
+
+    public static final ConfigOption<String> STORAGE_ENGINE =
+            new ConfigOption<>(
+                    "mysql.engine",
```

Comment:
prefer to rename to `jdbc.storage_engine` to keep compatible with other cloud databases based on postgre